core/shared/src/test/scala/fs2/StreamSuite.scala

```diff
@@ -453,6 +454,20 @@ class StreamSuite extends Fs2Suite {
     }
   }
 
+  group("translate") {
```

Comment:
Note there's a `StreamTranslateSuite` - maybe move this to that file?

Reply:
This is not strictly related to the main changes... reverted